test/functional/Valhalla/playlist.xml

```diff
@@ -27,6 +27,7 @@
 		<variations>
 			<variation>NoOptions</variation>
 			<variation>-Xgcpolicy:nogc -XX:ValueTypeFlatteningThreshold=99999</variation>
+			<variation>-Xgcpolicy:optthruput -Xnocompactgc -XX:ValueTypeFlatteningThreshold=99999</variation>
```

Comment:
@tajila I think it makes sense to enable the GC as a separate variation.  It may help track down if the issue is GC specific. What do you think?

Reply:
that sounds good